sections/03_categorize.md

```diff
@@ -167,6 +167,35 @@ advances to practical game-changers.
 Additionally, unique barriers exist in this space that may hinder progress in
 this field.
 
+###### Lack of ground-truth labels in data, and high cost in post-study validation by clinician experts.
+
+*Lack of true labels is perhaps among the biggest obstacles for electronic 
+health records(EHR)-based learing studies (e.g., phenotyping). Popular deep 
+learning (and machine learning) methods are mostly classification problem, 
+which require ground-truth labels (i.e., response variables) for training. 
+For EHR, unfortunately, this means we need to hire multiple expert clinicians 
+to manually read and annotate individual patients' records to assign "true" 
+labels (i.e., expert chart review). Sometimes, the learnt features also need to 
+be manually validated and interpreted by clinicians. This can be quite costly 
+in time and money [doi: 10.1016/j.ijmedinf.2016.09.014]. Due to resource limitations, 
+many existing works in this direction (including many referenced in this review) 
```

Comment:
Can you reference a few of them here? I think #25 falls into this. Might be nice to at least provide a quick group so that readers can get there.